cypress/helpers/TimeLoggingPage.js

```diff
@@ -0,0 +1,17 @@
+export default {
+    visit: () => cy.visit('/time-logging'),
+
+    getCalendar: () => cy.get('.calendar'),
+
+    getPageTitle: () => cy.get('.page__title'),
+
+    getTileForm: () => cy.get('.tile.form'),
+
+    getActiveTab: () => cy.get('.main-nav__link--active'),
+
+    getUserName: () => cy.get('.user-info__title'),
+
+    getTabs: () => cy.get('.main-nav').find('li'),
+
+    getCurrDay: () => cy.get('.calendar--today > span')
```

Comment:
why you are not using above approach here as well? 
 getTabs: () => cy.get('.main-nav').find('li'),